packages/protobuf/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@penumbra-zone/protobuf",
+  "version": "0.0.0",
+  "license": "(MIT OR Apache-2.0)",
+  "description": "Exports a `@bufbuild/protobuf` type registry with all message types necessary to communicate with a Penumbra extension",
+  "type": "module",
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "rm -rfv dist",
+    "lint": "eslint src",
+    "prepack": "pnpm clean && pnpm build"
+  },
+  "exports": {
+    ".": "./src/registry.ts"
+  },
+  "publishConfig": {
+    "exports": {
+      ".": {
+        "import": "./dist/registry.js",
+        "types": "./dist/registry.d.ts"
+      }
+    }
+  },
+  "bundleDependencies": [
+    "@buf/cosmos_ibc.bufbuild_es",
+    "@buf/cosmos_ibc.connectrpc_es",
+    "@buf/penumbra-zone_penumbra.bufbuild_es",
+    "@buf/penumbra-zone_penumbra.connectrpc_es"
+  ],
```

Comment:
this field directs npm to pack a `node_modules` with these dependencies in the emitted tarball. this means users should not have to configure the buf registry in order to use this package.